lib/config.yml

```diff
@@ -538,19 +365,18 @@ errorMessages:
     msg: No available transit routes or rideshare/carshare service at origin.
 
 reportIssue:
-  mailto: gis@trimet.org
+  mailto: hello@trimet.org
 
 analytics:
   google:
-    globalSiteTag: UA-133677388-1
+    globalSiteTag: UA-688646-7
```

Comment:
I don't think it'd be a good idea to use this google analytics key on our dev site. Can you change it back?